### ISSUE TITLE
serving static files: set a default filename extension

### ISSUE THREAD

**Issue by fghhfg** [lwsjs/local-web-server#120]
How do I serve webpage without extension (.html)?
Or it just a bad idea without .html on my static website?

It also doesn't work on `http-server` of npm, let's why I'm here...

ref:
https://github.com/indexzero/http-server/issues/160

**Comment by 75lb** [comment]
I have fixed this and released a prerelease. Could you test it works for you? 

Install the prerelease.

```
$ npm install -g local-web-server@next
```

Run `ws` with a default filename extension of `html`.

```
ws --static.extensions html
```

Let me know how it goes.

**Comment by 75lb** [comment]
Fixed and released in [v3.0.0](https://github.com/lwsjs/local-web-server/releases)
